docs/source/cloud-setup/cloud-permissions/aws.rst

```diff
@@ -1,39 +1,385 @@
+- If you use SSO login, enable some :ref:`additional cross-cloud features <sso-feature-compat>`.
+
+Follow these steps to create a new AWS user:
+
+1. Open the `IAM dashboard <https://us-east-1.console.aws.amazon.com/iamv2/home#/home>`_ in the AWS console and click on the **Users** tab.
+
+   .. image:: ../../images/screenshots/aws/aws-add-user.png
+       :alt: AWS Add User
+
+
+   Then, click **Create User** and enter the user's name. Click **Next**.
+
+2. In the **Permissions options** section, select "Attach existing policies directly". Depending on whether you want simplified or minimal permissions, follow the instructions below:
+
+   .. tab-set::
+
+       .. tab-item:: Simplified permissions
+
+           Search for the **AdministratorAccess** policy, and check the box to add it. Click **Next** to proceed.
+
+       .. tab-item:: Minimal permissions
+
+           Click on **Create Policy**.
+
+           .. image:: ../../images/screenshots/aws/aws-create-policy.png
+               :alt: AWS Create Policy
+
+           This will open a new window to define the minimal policy. Follow the instructions to create a new policy: :ref:`aws-minimal-policy`.
+
+           Come back to this window, and in the **Permissions policies** box, click on the refresh button. You can now search for the policy you just created.
+
+           .. image:: ../../images/screenshots/aws/aws-add-policy.png
+               :alt: AWS Add Policy
+
+           Check the box to add the policy, and click **Next** to proceed.
+
+3. Click on **Next** and follow the instructions to create the user.
+
+4. Select the newly created user from the dashboard, and go to the **Security credentials** tab. Click on **Create access key**.
+
+   .. image:: ../../images/screenshots/aws/aws-create-access-key.png
+       :alt: AWS Create access key
+
+5. For "Use case", select **Other**. Click **Next**, then click **Create access key**.
+
+6. Use the newly created access key to configure your credentials with the AWS CLI:
+
+   .. code-block:: console
+     :emphasize-lines: 13-14
+
+     $ # Configure your AWS credentials
+     $ aws configure
+     AWS Access Key ID [None]: <Access key>
+     AWS Secret Access Key [None]: <Secret access key>
+     Default region name [None]:
+     Default output format [None]:
+
+     $ # Check that AWS sees the shared-credentials-files
+     $ aws configure list
+           Name                    Value             Type    Location
+           ----                    -----             ----    --------
+        profile                <not set>             None    None
+     access_key     ****************xxxx shared-credentials-file
+     secret_key     ****************xxxx shared-credentials-file
+         region                <not set>             None    None
+
+     $ # Validate that credentials are working
+     $ sky check aws -v
+
+
+.. _several-aws-profiles:
+
+Switch profiles or accounts
```

Comment:
Pull this up to 2nd subsection? Since this one is lower-lift, potentially user-facing.

How about:
- Using AWS SSO
- Using a specific VPC
- Switching profiles or accounts
- <the rest, admin-facing>

Reply:
Hmm, I feel that "Using a specific VPC" is probably more admin-facing than "Dedicated SkyPilot IAM user". Anyway, I'm pretty happy with any order as long as SSO is at the top.

Reply:
Reordered:

- Using AWS SSO
  - Multi-cloud access with SSO login
- Switching profiles or accounts
- Using a specific VPC
- Dedicated SkyPilot IAM user
- Minimal permissions
  - Create a user account with minimal permissions
  - Create the internal IAM role for SkyPilot
  - Minimal IAM policy
- Troubleshooting
  - Common issues

@concretevitamin what do you think?

Reply:
Sounds good.